docs/prover/approx/summarization.md

```diff
@@ -27,9 +23,12 @@ methods {
 }
 
 // Example: Summarization for a complex function
+function multiply(uint256 x, uint256 y) returns uint256 {
+    return x * y;
+}
 methods {
     complexFunction(uint256 x, uint256 y) returns bool =>
-        EXISTS uint256 z . z == x * y;
+        EXISTS uint256 z . z == multiply(x, y);
```

Comment:
I still don't understand how this compiles with the all-caps EXISTS

Reply:
Done